addons/event/static/src/client_action/event_registration_summary_dialog.xml

```diff
@@ -59,18 +59,23 @@
                     </table>
                 </div>
             </div>
-            <div class="d-flex flex-column align-items-start">
-                <span class="fs-2 btn btn-link ps-0" role="button" t-on-click="() => this.onRegistrationPrintPdf()">
-                    <i class="fa fa-solid fa-print"/><span > Print Ticket</span>
-                </span>
-                <span class="fs-2 btn btn-link ps-0" role="button" t-on-click="() => this.onRegistrationView()">
-                    <i class="fa fa-external-link"/><span> Open Details</span>
-                </span>
+            <div class="d-flex justify-content-between">
+                <select t-if="registration.iot_printers.length > 1" type="select" class="form-select w-75" t-model.number="printSettings.iotPrinterId" t-on-change="savePrintSettings">
+                    <option value="" disabled="true" t-att-selected="!printSettings.iotPrinterId">Select printer...</option>
+                    <option t-foreach="registration.iot_printers" t-as="printer" t-key="printer.id" t-att-value="printer.id">
+                        <t t-esc="printer.name" />
+                    </option>
+                </select>
+                <div class="form-check">
+                    <input id="autoprint" type="checkbox" class="form-check-input" t-model="printSettings.autoPrint" t-on-change="savePrintSettings"></input>
+                    <label for="autoprint" class="form-check-label">Auto-Print</label>
```

Comment:
this is currently always visible, even if no iot module is installed. we should make it invisible just as the printer selection secion